src/renderer/pages/VideoEditorPage.vue

```diff
@@ -86,9 +90,20 @@ import { join } from 'path';
 import VideoPlayer from '../components/VideoPlayer.vue'
 import { generateMetronome } from '../util/metronome'
 
+
+const thumbnailFrequency = '1/5';
```

Comment:
This should be calculated dynamically based on thumbnail container width/height, output aspect ratio and video length